src/etl/LoadBalancer.cpp

```diff
@@ -77,13 +77,18 @@ LoadBalancer::LoadBalancer(
     SourceFactory sourceFactory
 )
 {
-    auto const forwardingCacheTimeout = config.valueOr<float>("forwarding_cache_timeout", 0.f);
+    auto const forwardingCacheTimeout = config.valueOr<float>("forwarding.cache_timeout", 0.f);
     if (forwardingCacheTimeout > 0.f) {
         forwardingCache_ = impl::ForwardingCache{std::chrono::milliseconds{
             std::lroundf(forwardingCacheTimeout * static_cast<float>(util::MILLISECONDS_PER_SECOND))
         }};
     }
 
+    auto const forwardingXUserValue = config.valueOr<std::string>("forwarding.x_user_value", {});
+    if (not forwardingXUserValue.empty()) {
```

Comment:
These kind of ifs can be collapsed to `if (auto const value = ...; not value.empty()) ...` if you don't need `forwardingXUserValue` down the line.

Reply:
or even directly assign `config.maybeValue` to the member if that produces the same effect.